hangman_game.rb

```diff
@@ -0,0 +1,89 @@
+
+require 'net/http'
+require 'uri'
+require 'json'
+
+def choose_random_word
+  url = URI.parse("https://random-word-api.herokuapp.com/word?number=10")
+
+  http = Net::HTTP.new(url.host, url.port)
+  http.use_ssl = (url.scheme == "https")
```

Comment:
remove brackets

Reply:
In this case, the URL scheme will always be 'https', so you can directly set use_ssl true.